packages/fetch-engine/src/download.ts

```diff
@@ -258,7 +257,7 @@ async function binaryNeedsToBeDownloaded(
     return false
   }
   // 1. Check if file exists
-  const targetExists = await exists(job.targetFilePath)
+  const targetExists = fs.existsSync(job.targetFilePath)
```

Comment:
`fs.exists` is deprecated since `v1.0.0`
https://nodejs.org/api/fs.html#fsexistspath-callback